PermanentOrg/web-app
# Per 8562 indicate fields that are editable

Steps to test:
1.Open any uploaded file in the full screen viewer.
2. Notice that an icon and Click to add text is present for tags and location (description already had a helper text and the name cannot be deleted.)
